aws-iot-jobtemplate/aws-iot-jobtemplate.json

```diff
@@ -157,7 +157,7 @@
     }
   },
   "properties": {
-    "JobTemplateArn": {
+    "Arn": {
```

Comment:
Why are we updating the key names here? It looks unnecessary. Also `JobTemplateArn` is a better key name than just `Arn` .

Reply:
Arn is the proper naming convention for a CFN resource. Like when calling !GetAtt LogicalIdentifier.Arn -- I only realized this when writing these test templates.